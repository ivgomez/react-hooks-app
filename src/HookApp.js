import React from "react";

export const HookApp = () => {
  return (
    <div>
      <h1>Hello react</h1>
    </div>
  );
};
