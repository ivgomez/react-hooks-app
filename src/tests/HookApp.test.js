import React from "react";
import "@testing-library/jest-dom";
import { shallow } from "enzyme";
import { HookApp } from "../HookApp";

describe("Test on <HookApp>", () => {
  test("it should rendered ok", () => {
    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
