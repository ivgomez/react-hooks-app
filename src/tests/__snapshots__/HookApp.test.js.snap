// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on <HookApp> it should rendered ok 1`] = `
<div>
  <h1>
    Hello react
  </h1>
</div>
`;
